CHANGELOG.md

```diff
@@ -5,6 +5,11 @@
   * Make `shippingMethod` property on `BTThreeDSecureRequest` an enum instead of a string
   * Remove `BTTokenizationService`
   * Make `BTPaymentMethodNonceParser` private
+  * Remove `BTAppSwitchDelegate`
+  * Rename `BTAppSwitch` to `BTAppContextSwitcher`
```

Comment:
We tried to come up with a name that would indicate that this class handles app switches as well as browser switches (i.e., returning from `SFSafariViewController`). If anyone has other suggestions, we'd love to hear them!

Reply:
`BTSwitchAllTheThings` 🤣  tbh I think what you have is the best. Just spent five minutes seriously considering other names and I don't like any of my ideas